person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Let's add the `@id` instance variable and set it to a random number so that each new person created has a unique `id`. 

```suggestion
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
  end
```